packages/qwik-city/buildtime/vite/dev-server.ts

```diff
@@ -28,6 +28,11 @@ export function ssrDevMiddleware(ctx: BuildContext, server: ViteDevServer) {
       const url = new URL(req.originalUrl!, `http://${req.headers.host}`);
       const pathname = url.pathname;
 
+      if (isViteInternalRequest(pathname)) {
```

Comment:
I think safely check if `req.headers.accept.includes('text/html')` then continue otherwise go next should be fine?

Reply:
nvm. I was wrong. city can handling other type requests too.